ansible/tasks/internal/supautils.yml

```diff
@@ -60,6 +60,28 @@
     state: present
     line: supautils.placeholders_disallowed_values = 'content-type'
 
+- name: supautils - set supautils.privileged_extensions
+  become: yes
+  lineinfile:
+    path: /etc/postgresql/postgresql.conf
+    state: present
+    # full list: supautils.privileged_extensions = 'address_standardizer, address_standardizer_data_us, adminpack, amcheck, autoinc, bloom, btree_gin, btree_gist, citext, cube, dblink, dict_int, dict_xsyn, earthdistance, file_fdw, fuzzystrmatch, hstore, http, insert_username, intagg, intarray, isn, lo, ltree, moddatetime, old_snapshot, pageinspect, pg_buffercache, pg_cron, pg_freespacemap, pg_graphql, pg_hashids, pg_net, pg_prewarm, pg_stat_statements, pg_surgery, pg_trgm, pg_visibility, pgaudit, pgcrypto, pgjwt, pgrouting, pgrowlocks, pgsodium, pgstattuple, pgtap, plcoffee, pljava, plls, plpgsql, plpgsql_check, plv8, postgis, postgis_raster, postgis_sfcgal, postgis_tiger_geocoder, postgis_topology, postgres_fdw, refint, rum, seg, sslinfo, supautils, tablefunc, tcn, tsm_system_rows, tsm_system_time, unaccent, uuid-ossp'
+    #
+    # omitted because may be unsafe: adminpack, amcheck, file_fdw, lo, old_snapshot, pageinspect, pg_buffercache, pg_freespacemap, pg_prewarm, pg_surgery, pg_visibility, pgstattuple
+    #
+    # omitted because doesn't require superuser: pgjwt, pgtap, postgis_tiger_geocoder
+    #
+    # omitted because deprecated: intagg
+    #
+    line: supautils.privileged_extensions = 'address_standardizer, address_standardizer_data_us, autoinc, bloom, btree_gin, btree_gist, citext, cube, dblink, dict_int, dict_xsyn, earthdistance, fuzzystrmatch, hstore, http, insert_username, intarray, isn, ltree, moddatetime, pg_cron, pg_graphql, pg_hashids, pg_net, pg_stat_statements, pg_trgm, pgaudit, pgcrypto, pgrouting, pgrowlocks, pgsodium, plcoffee, pljava, plls, plpgsql, plpgsql_check, plv8, postgis, postgis_raster, postgis_sfcgal, postgis_topology, postgres_fdw, refint, rum, seg, sslinfo, supautils, tablefunc, tcn, tsm_system_rows, tsm_system_time, unaccent, uuid-ossp'
```

Comment:
Feel like things would be cleaner if we moved these "add line to file" or "run a regexp replace on the file" if we just moved these lines to the source template, with appropriate guards on the internal template var being defined

Reply:
Do you mean in `postgresql.conf.j2`? I can make a separate PR for it

Reply:
Yeah